<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>faceDetect</class>
 <widget class="QWidget" name="faceDetect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>824</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 31, 32);</string>
  </property>
  <widget class="QPushButton" name="faceHaar">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>70</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>人脸识别</string>
   </property>
  </widget>
  <widget class="QPushButton" name="takePic">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>拍照</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>430</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:30px;</string>
   </property>
   <property name="text">
    <string>开始流</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>320</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <item>
    <property name="text">
     <string>        设备</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="displayLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>621</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>src</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="closeDetect">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>关闭人脸识别</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
